### ISSUE TITLE
Plan card styling

### ISSUE THREAD

**Issue by jpatrici** [chingu-voyages/v39-bears-team-16#96]
Should be done after progress bar and button update is done
- Need to add likes functionality (will be in another ticket)
- Add share functionality (will be in another ticket and should be done after #125)
- progress bar
- Editor: add public/private icon beside plan name

![Screen Shot 2022-07-29 at 3.19.54 PM.png](https://images.zenhubusercontent.com/625a22079da9ed3cd6dee70d/41271f21-2a11-4878-8973-5657eae954e0)
